gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -722,6 +727,36 @@ gst_tensor_filter_get_property (GObject * object, guint prop_id,
     case PROP_CUSTOM:
       g_value_set_string (value, prop->custom_properties);
       break;
+    case PROP_SUBPLUGINS:
+    {
```

Comment:
Does it actually do something except gst-inspect?

Reply:
this property is just to print sub-plugins list.